config/initializers/raven.rb

```diff
@@ -7,6 +7,7 @@
 
 Raven.configure do |config|
   config.dsn = Settings.sentry.dsn if Settings.sentry.dsn
+  config.environment = Settings.vsp_environment if Settings.vsp_environment
```

Comment:
Is `~` an acceptable value here?  Lighthouse had some things break because somebody put in a filler value in settings that broke the development and staging deployments that didn't have the right `settings.local.yml` to go with it